config/i18n.ts

```diff
@@ -20,16 +20,6 @@ const locales: LocaleObjectData[] = [
     file: 'en-GB.json',
     name: 'English (UK)',
   },
-  ({
-    code: 'ar-EG',
```

Comment:
`ar-EG` should be at top, below `en` variants, do not modify this, read contributing guide

Reply:
Fixed! Just committed the file updated.